test/functional/tests/cli/test_cli_standby.py

```diff
@@ -297,3 +297,48 @@ def test_start_neg_cli_flags():
                     f'"{mutually_exclusive_params_load[0]}"'
                     f'Got "{output.stderr}" instead.'
                 )
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+def test_activate_without_detach():
+    """
+    title: Activate cache without detach command.
+    description: |
+       Try activate passive cache without detach command before activation.
+    pass_criteria:
+      - The activation is not possible
+      - The cache remains in Standby state after unsuccessful activation
+      - The cache exported object is present after an unsuccessful activation
```

Comment:
The test doesn't verify if the cache exported object is present after an unsuccessful activation

Reply:
Right, I moved check after cache activation.

Reply:
Actually, I think there should be two checks - before and after the unsuccessful activation.

Reply:
And one more thing; I know that the test is verifying the activate exit status and the error message but there should be an additional check to make sure that the cache is still running in standby mode

Reply:
Good point. Now exported object presence is checked after load and unsuccessful activate.
After activation cache state is also checked.